pkg/helmraiser/helm.go

```diff
@@ -0,0 +1,147 @@
+package helmraiser
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	jsonnet "github.com/google/go-jsonnet"
+	"github.com/google/go-jsonnet/ast"
+	"github.com/pkg/errors"
+	yaml "gopkg.in/yaml.v3"
+)
+
+var deferTempFiles []string
+
+func confToArgs(conf map[string]interface{}, args *[]string) error {
```

Comment:
This function should be self-contained and not modify outside state. To do so, accept `conf`, do create the files needed, but return their names, and also return the extracted flag array.

The consumer of this function should be responsible for adding this information to the appropriate places